### PR TITLE
Added BowerJsonExtractor + Tests

### PR DESCRIPTION
**Pull request by albert7c**

### CODE REVIEW

**Comment by albert7c** [comment]
Screen Shot 2019-07-11 at 10 21 44 PM

**Comment by mjpitz** [comment]
@albert7c : If you do `npm test -- -u` locally and commit the generated snapshots, then it should resolve this issue.

File: src/extractors/BowerJsonExtractor.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+        const { organization, module } = parseName(name);
+
+        let allDependencies = extract((dependencies || {}), "");
+        allDependencies = allDependencies.concat(extract((devDependencies || {}), "dev"));
+        allDependencies = allDependencies.concat(extract((bundledDependencies || {}), "bundled"));
+
+        return {
+            language: Languages.NODE,
+            system: "npm",
```
Comment (mjpitz) [inline src/extractors/BowerJsonExtractor.ts]: this should be `bower`

File: src/extractors/BowerJsonExtractor.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            organization, module, version,
```
Comment (mjpitz) [inline src/extractors/BowerJsonExtractor.ts]: one per line

**Comment by mjpitz** [comment]
Only other change is to update the registry with an entry for this new extractor: https://github.com/deps-cloud/extractor/blob/master/src/extractors/ExtractorRegistry.ts

**Comment by albert7c** [comment]
Screen Shot 2019-07-13 at 8 09 00 AM

**Comment by albert7c** [comment]
Although.. when i ran the npm test -- -u command, 
I got jest command not found

**Comment by mjpitz** [comment]
@albert7c : Merged! Thanks for the contribution.

**Pull request merged**